Makefile

```diff
@@ -78,19 +79,19 @@ install:
 
 srpm: source
 	if test ! -d BUILD/SRPM; then mkdir -p BUILD/SRPM; fi
-	mock --resultdir BUILD/SRPM -D "commit $(COMMIT)" --buildsrpm --spec avocado.spec --sources SOURCES
+	mock -r $(MOCK_CONFIG) --resultdir BUILD/SRPM -D "commit $(COMMIT)" --buildsrpm --spec avocado.spec --sources SOURCES
```

Comment:
Although what do you use to configure `aexpect` repo? The default works well for me, but the `make MOCK_CONFIG=epel-6-x86_64 rpm` fails. Anyway it's a configuration issue, not really related to this, so merging...